### PR TITLE
Feature start time

### PR DESCRIPTION
**Pull request by ghost**
Made the base for the TimeManager class which will controll the RTC module.
Also made a start on the RTCTime struct which is a struct that contains information about time and dates.

### CODE REVIEW

**Comment by DanielvanVliet** [comment]
Looks good, ready for merge

File: src/rtc_time.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+uint8_t RTCTime::getYear() {
+    return year;
+}
+
+
+
+void RTCTime::setSeconds(uint8_t seconds) {
+    this->seconds = seconds;
```
Comment (BIGduzy) [inline src/rtc_time.cpp]: There is no need for the this pointer here, you can just use `seconds = seconds`.
The same goes for the rest of the file.

**Review by BIGduzy — CHANGES_REQUESTED** [review]
Please remove the use of the `this` pointer where not necessary.

File: src/rtc_time.cpp [hunk 2]
```diff
@@ -7,84 +7,117 @@
...
+    seconds = seconds;
+    minutes = minutes;
+    hours = hours;
+    dayOfTheWeek = dayOfTheWeek;
+    dayOfTheMonth = dayOfTheMonth;
+    month = month;
+    year = year;
+}
+
+bool operator< (const RTCTime& lhs, const RTCTime& rhs) {
```
Comment (BIGduzy) [inline src/rtc_time.cpp]: o.0

You should be able to do this in a better way, remove those sneaky returns and don't nest that deep

**Review by BIGduzy — APPROVED** [review]

**Pull request merged**